src/sagas/languageSagas.js

```diff
@@ -0,0 +1,21 @@
+import { call, takeLatest, put } from 'redux-saga/effects';
+
+import { LANGUAGE } from 'constants/actionConstants';
+import fetchLanguages from 'apis/languagesApis';
+import { showErrorMessage } from 'actions/userDriveActions';
+import { setLanguages } from 'actions/languageAction';
+
+// worker saga
+export function* languageDetails(action) {
+  try {
+    const response = yield call(fetchLanguages);
+    yield put(setLanguages(response.data));
+  } catch (err) {
+    yield put(showErrorMessage(err.message));
+  }
+}
+
+// watcher saga
+export default function* languageSagas() {
```

Comment:
Codacy has a fix for the issue: [Missing space before *.](https://app.codacy.com/gh/joshsoftware/OnCoT-Frontend/pullRequest?prid=7073195)
```suggestion
export default function * languageSagas() {
```